tests/integration/awslambda/test_lambda_api.py

```diff
@@ -4884,7 +4884,8 @@ def test_layer_policy_lifecycle(
 @pytest.mark.skipif(condition=is_old_provider(), reason="not supported")
 class TestLambdaSnapStart:
     @pytest.mark.aws_validated
-    def test_snapstart_lifecycle(self, create_lambda_function, snapshot, aws_client):
+    @pytest.mark.parametrize("runtime", [Runtime.java11, Runtime.java17])
```

Comment:
good parametrization 👍 

(🔮 if we automatically re-generate snapshots in the future, we could consider choosing only one runtime if it still takes ~2min against AWS.)

Reply:
:+1: There are a couple test where we would need to skip that in the lambda tests, most notably all the concurrency stuff.